controllers/replicaset_controller_test.go

```diff
@@ -87,6 +87,15 @@ func newScramReplicaSet(users ...mdbv1.MongoDBUser) mdbv1.MongoDBCommunity {
 }
 
 func newTestReplicaSetWithTLS() mdbv1.MongoDBCommunity {
+	return newTestReplicaSetWithTLSCaCertificateReferences(&mdbv1.LocalObjectReference{
```

Comment:
Could you please help me understand what this change bring to the table? I'm not sure if it changes anything?

Reply:
It allows us to create test fixtures, where the caller passes custom arguments.